flask-app/BussIT-API/main.py

```diff
@@ -13,7 +13,7 @@ def vasttrafik(id=None):
             else 200
 
         return jsonify(response), status,\
-            {'Access-Control-Allow-Origin': '*'}
+            {'Access-Control-Allow-Origin': 'http://localhost:3000'}
```

Comment:
Tänkte mer på ”proxy”: ”http://localhost:3000” i package.json

https://create-react-app.dev/docs/proxying-api-requests-in-development/

Reply:
I have never done this before, so ill put this up as an issue